NorthstarDedicatedTest/serverauthentication.cpp

```diff
@@ -212,7 +213,7 @@ bool ServerAuthenticationManager::AuthenticatePlayer(void* player, int64_t uid,
 		{
 			authFail = false;
 			// uuid
-			strcpy((char*)player + 0xF500, strUid.c_str());
+			strcpy((char*)player + 0xF00, strUid.c_str());
```

Comment:
```suggestion
			strcpy((char*)player + OFFSET_PLAYER_UID, strUid.c_str());
```
Missed/typoed one ;)

Reply:
forgor 💀